ci/install_llvm.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Download, build and install LLVM from source
```

Comment:
This file isn't used anymore? Should we leave it out until there's a need for it?

Reply:
Removed